config/routes.rb

```diff
@@ -1,6 +1,31 @@
 Rails.application.routes.draw do
+  mount Rswag::Ui::Engine => '/api-docs'
+  mount Rswag::Api::Engine => '/api-docs'
+  get 'root/index'
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
   # Defines the root path route ("/")
   # root "articles#index"
+  root 'root#index'
+
+  namespace :api do
+    namespace :v1 do 
+      resources :hotels, only: [:index]
+      resources :users do 
+        resources :reservations, only: [:index]    
+        resources :hotels, only: [:index, :show] do 
+          resources :reservations, only: [:create, :destroy]  
+        end    
+      end
+    end
+  end
+
+  # namespace :api do
+  #   namespace :v1 do
+  #     resources :users do
+        
+  #     end
+  #   end
+  # end
```

Comment:
- [ ] Please, delete this old piece of commented code to improve the cleanliness of your repo.  Also, kindly remove the additional commented code present in this and the following files:

1. hotel.rb
2. user.rb
3. application.html.erb
4. users_spec.rb
5. ReserveDetails.js
6. HomePageAction.js
7. HotelAction.js
8. App.css

That way, your repo's quality will improve a lot 💪